### PR TITLE
docs: add explanation on crud usage

### PR DESCRIPTION
**Pull request by cristianmtr**
This is required in order to explain what a Storage vs Searcher implements (generally).

### CODE REVIEW

File: docs/advanced/experimental/indexers.md [hunk 1]
```diff
@@ -39,6 +39,22 @@ Besides, there are two types of special indexer,
...
+
+| Operation  | Endpoint  | Implemented in |
+|------------|-----------|----------------|
+| **C**reate | `/index`  | Storage        |
+| **R**ead   | `/search` | Searcher       |
+| **U**pdate | `/update` | Storage        |
+| **D**elete | `/delete` | Storage        |
+
+The Create, Update, Delete are implemented by the Storage Indexers, while the Read operation is implemented in the `/search` endpoints in the Search Indexers. 
+The `/search` endpoints do not correspond perfectly with the Read operation, as it _searches_ for similar results, and not return a specific Document by id.
```
Comment (maximilianwerk) [inline docs/advanced/experimental/indexers.md:54]: So actually, we have a SCUD interface? :crab:

**Review by maximilianwerk — DISMISSED** [review]
If the linking happens in the next step: LGTM

File: docs/advanced/experimental/indexers.md (hunk 1 above)
Comment (cristianmtr) [reply docs/advanced/experimental/indexers.md:54]: It depends on how we want to frame this: 

> Read is search

or

> Read is fill_embedding, with Search being its own primary operation

File: docs/advanced/experimental/indexers.md [hunk 2]
```diff
@@ -39,6 +39,22 @@ Besides, there are two types of special indexer,
 
 ```
 
+## CRUD operations and the Executor Endpoints
```
Comment (hanxiao) [inline docs/advanced/experimental/indexers.md:42]:
```suggestion
## CRUD operations and the Executor endpoints
```

besides h1, all titles should only uppercase the first letter

**Review by hanxiao — DISMISSED** [review]

**Comment by cristianmtr** [comment]
@hanxiao Headers need to be in Title Case, afaik. https://en.wikipedia.org/wiki/Title_case

Also, the commit message is invalid

File: docs/advanced/experimental/indexers.md [hunk 3]
```diff
@@ -39,6 +39,22 @@ Besides, there are two types of special indexer,
...
+
+| Operation  | Endpoint  | Implemented in |
+|------------|-----------|----------------|
+| **C**reate | `/index`  | Storage        |
+| **R**ead   | `/search` | Searcher       |
+| **U**pdate | `/update` | Storage        |
+| **D**elete | `/delete` | Storage        |
+
+The Create, Update, Delete are implemented by the Storage Indexers, while the Read operation is implemented in the `/search` endpoints in the Search Indexers. 
+The `/search` endpoints do not correspond perfectly with the Read operation, as it _searches_ for similar results, and not return a specific Document by id.
```
Comment (alexcg1) [inline docs/advanced/experimental/indexers.md:54]: - and **does** not return...

File: docs/advanced/experimental/indexers.md [hunk 4]
```diff
@@ -39,6 +39,22 @@ Besides, there are two types of special indexer,
...
+The Executors implemented and provided by Jina implement a CRUD interface as follows:
+
+| Operation  | Endpoint  | Implemented in |
+|------------|-----------|----------------|
+| **C**reate | `/index`  | Storage        |
+| **R**ead   | `/search` | Searcher       |
+| **U**pdate | `/update` | Storage        |
+| **D**elete | `/delete` | Storage        |
+
+The Create, Update, Delete are implemented by the Storage Indexers, while the Read operation is implemented in the `/search` endpoints in the Search Indexers.
```
Comment (alexcg1) [inline docs/advanced/experimental/indexers.md:53]: - The C, U, D **operations**

File: docs/advanced/experimental/indexers.md [hunk 5]
```diff
@@ -39,6 +39,22 @@ Besides, there are two types of special indexer,
...
+| Operation  | Endpoint  | Implemented in |
+|------------|-----------|----------------|
+| **C**reate | `/index`  | Storage        |
+| **R**ead   | `/search` | Searcher       |
+| **U**pdate | `/update` | Storage        |
+| **D**elete | `/delete` | Storage        |
+
+The Create, Update, Delete are implemented by the Storage Indexers, while the Read operation is implemented in the `/search` endpoints in the Search Indexers. 
+The `/search` endpoints do not correspond perfectly with the Read operation, as it _searches_ for similar results, and not return a specific Document by id.
+Some Storage Indexers do implement a `/fill_embedding` endpoint, which function as a Read by id.
```
Comment (alexcg1) [inline docs/advanced/experimental/indexers.md:55]: - which function**s**

**Review by alexcg1 — CHANGES_REQUESTED** [review]
Just a few minor tweaks. Otherwise LGTM :+1:

**Comment by cristianmtr** [comment]
@alexcg1 https://github.com/jina-ai/jina/pull/3490/files

**Pull request merged**